docs/product-getting-started-guide/src/main/asciidoc/business-rules-create-proc.adoc

```diff
@@ -0,0 +1,73 @@
+[id='guided-rule-licence-suspension-create-proc']
+= Creating the Driver License Suspension rule
+
+The following chapter instructs you how to create a driver license suspension rule in {PRODUCT} using the Guided Rule wizard.
+
+[float]
+== Prerequisite
+
+Created both the Violation and Driver data objects.
+
+.Procedure
+. Log in to {CENTRAL}. Click *Menu* -> *Design* -> *Projects*, then *Driver department traffic violations*.
+. Click *Create New Asset* -> *Guided Rule*, then enter:
+
+ * *Guided Rule*: `DriverLicenseSuspensionRule`
+ * *Package*: `org.jboss.example.traffic_violations`
++
+
+. Click *+Ok* to open the *Guided Rule designer*.
++
+
+.Create new Guided Rule window
+image::guided-rule.png[]
+
+== Setting the Suspension rule conditions
```

Comment:
See my comment in the Adding the Violation data object data fields section, about intro statements.

Reply:
done